easybuild/easyconfigs/f/FLTK/FLTK-1.3.3-intel-2016a-libX11-1.6.3.eb

```diff
@@ -0,0 +1,43 @@
+#
+# author: Dina Mahmoud Ibrahim ( Cairo University )
+#
+easyblock = 'ConfigureMake'
+
+name = 'FLTK'
+version = '1.3.3'
+
+libx11 = 'libX11'
+libxver = '1.6.3'
+versionsuffix = '-%s-%s' % (libx11, libxver)
+
+homepage = 'http://www.fltk.org'
+description = """FLTK is a cross-platform C++ GUI toolkit for UNIX/Linux (X11), Microsoft Windows,
+ and MacOS X. FLTK provides modern GUI functionality without the bloat and supports 3D graphics via OpenGL
+ and its built-in GLUT emulation."""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+toolchainopts = {'optarch': True, 'pic': True}
+
+sources = ['%(namelower)s-%(version)s-source.tar.gz']
+source_urls = ['http://fltk.org/pub/%(namelower)s/%(version)s/']
+
+
+dependencies = [
+    (libx11, libxver),
+    ('libXcursor', '1.1.14'),
+    ('libXinerama', '1.1.3'),
+    ('Tcl', '8.6.4'),
```

Comment:
are you sure bumping from `8.5.x` to `8.6.x` is OK? It's not backwards compatible.